tests/integration-test.sh

```diff
@@ -22,19 +22,27 @@ dpkg -l > package.list
 
 
 buildinfo=$(drone build info vmware/vic $DRONE_BUILD_NUMBER)
+prNumber=$(drone build info --format "{{ .Ref }}" vmware/vic $DRONE_BUILD_NUMBER | cut -f 3 -d'/')
+prBody=$(curl https://api.github.com/repos/vmware/vic/pulls/$prNumber | jq -r ".body")
+
+if (echo $prBody | grep -q "\[fast fail\]"); then
+    export FAST_FAILURE=1
+else
+    export FAST_FAILURE=0
+fi
 
 if [[ $DRONE_BRANCH == "master" || $DRONE_BRANCH == "releases/"* ]] && [[ $DRONE_REPO == "vmware/vic" ]] && [[ $DRONE_BUILD_EVENT == "push" ]]; then
     echo "Running full CI for $DRONE_BUILD_EVENT on $DRONE_BRANCH"
     pybot --removekeywords TAG:secret --exclude skip tests/test-cases
 elif [[ $DRONE_BRANCH == *"refs/tags"* ]] && [[ $DRONE_REPO == "vmware/vic" ]] && [[ $DRONE_BUILD_EVENT == "tag" ]]; then
     echo "Running only Group11-Upgrade and 7-01-Regression for $DRONE_BUILD_EVENT on $DRONE_BRANCH"
     pybot --removekeywords TAG:secret --suite Group11-Upgrade --suite 7-01-Regression tests/test-cases
-elif grep -q "\[full ci\]" <(drone build info vmware/vic $DRONE_BUILD_NUMBER); then
+elif (echo $prBody | grep -q "\[full ci\]"); then
```

Comment:
Any chance we could have these match for the following for example:
- [x] <!-- CI-DIRECTIVE:full ci -->run full CI
- [x] <!-- CI-DIRECTIVE:unit tests -->run unit tests
- [x] <!-- CI-DIRECTIVE:Group23-VIC-Machine-Service -->run group23
```
- [x] <!-- CI-DIRECTIVE:full ci -->run full CI
- [x] <!-- CI-DIRECTIVE:unit tests -->run unit tests
- [x] <!-- CI-DIRECTIVE:Group23-VIC-Machine-Service -->run group23
```
This would make it quick and easy to switch between multiple groups or focused groups/tests.
Admittedly most PRs won't really benefit that much from it, but I would have found it really useful for my most recent few.

I'm envisaging some staple ones added to the template, and then provision for customization